ChangeLog.md~

```diff
@@ -0,0 +1,150 @@
+v1.7.2 / 2018-02-20
```

Comment:
Whoops! Looks like you checked in an editor backup file, might want to add `*~` to `.gitignore` and removing this.